packages/component/src/SendBox/ConnectivityStatus.js

```diff
@@ -89,11 +90,16 @@ export default connectConnectivityStatus(
               <SpinnerAnimation />
               { localize('INITIAL_CONNECTION_NOTIFICATION', language) }
             </React.Fragment>
-          : connectivityStatus === 'reconnecting' &&
+          : connectivityStatus === 'reconnecting' ?
```

Comment:
chaining tertiary operators together seems like a less straight-forward pattern to me than a switch statement, but that's a personal preference

Reply:
yeah, I had it as switch statements previously but was asked to change it lol. For simplicity I'm going to keep it as is.